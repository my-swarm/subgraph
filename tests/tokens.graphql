query {
  tokens {
    id
    owner
    address
    symbol
  }
}
