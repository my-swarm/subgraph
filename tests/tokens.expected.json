{
  "data": {
    "tokens": [
      {
        "address": "0x71c95911e9a5d330f4d621842ec243ee1343292e",
        "id": "0x71c95911e9a5d330f4d621842ec243ee1343292e",
        "owner": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
        "symbol": "TT2"
      },
      {
        "address": "0x7290f72b5c67052dde8e6e179f7803c493e90d3f",
        "id": "0x7290f72b5c67052dde8e6e179f7803c493e90d3f",
        "owner": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
        "symbol": "TT5"
      },
      {
        "address": "0x8464135c8f25da09e49bc8782676a84730c318bc",
        "id": "0x8464135c8f25da09e49bc8782676a84730c318bc",
        "owner": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
        "symbol": "TT1"
      },
      {
        "address": "0x90118d110b07abb82ba8980d1c5cc96eea810d2c",
        "id": "0x90118d110b07abb82ba8980d1c5cc96eea810d2c",
        "owner": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
        "symbol": "TT3"
      },
      {
        "address": "0xafe1b5bdebd4ae65af2024738bf0735fbb65d44b",
        "id": "0xafe1b5bdebd4ae65af2024738bf0735fbb65d44b",
        "owner": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
        "symbol": "TT4"
      }
    ]
  }
}
